### PR TITLE
Resource relationships

### PR DESCRIPTION
**Pull request by EdwinHoksberg**
To access the relationship object in the responses

### CODE REVIEW

File: src/Resource/Invoice.php [hunk 1]
```diff
@@ -56,6 +56,11 @@ class Invoice extends AbstractResource
      */
     public $transaction_date;
 
+    /**
+     * @var mixed
```
Comment (tdgroot) [inline src/Resource/Invoice.php]:
```suggestion
     * @var \stdObject
```

File: src/Resource/Customer.php [hunk 2]
```diff
@@ -92,6 +92,10 @@ class Customer extends AbstractResource
      * @var string
      */
     public $language;
+    /**
+     * @var mixed
```
Comment (tdgroot) [inline src/Resource/Customer.php]:
```suggestion
     * @var \stdObject
```

File: src/Resource/Subscription.php [hunk 3]
```diff
@@ -70,6 +70,10 @@ class Subscription extends AbstractResource
      * @var mixed
      */
     public $metadata;
+    /**
+     * @var mixed
```
Comment (tdgroot) [inline src/Resource/Subscription.php]:
```suggestion
     * @var \stdObject
```

File: src/Resource/SubscriptionPlan.php [hunk 4]
```diff
@@ -41,6 +41,10 @@ class SubscriptionPlan extends AbstractResource
      * @var string
      */
     public $terms;
+    /**
+     * @var mixed
```
Comment (tdgroot) [inline src/Resource/SubscriptionPlan.php]:
```suggestion
     * @var \stdObject
```

**Review by tdgroot — CHANGES_REQUESTED** [review]
Thank you for your pull request! :+1: 

I have done some suggestions, please review them!

**Comment by EdwinHoksberg** [comment]
Thanks for your review! I applied your suggestions

**Pull request merged**